core/bindings.js

```diff
@@ -20,7 +20,7 @@ Deno.__op__registerOp = function (isAsync, op, opName) {
       return;
     }
     core.asyncOps[opName] = op;
-    core.ops[opName] = function (...args) {
+    const fn = function (...args) {
```

Comment:
We may want to test the performance of this, but you can name a function using this construct:

```
{ [name]() { ... } }[name]
```

Reply:
It's done only during snapshotting AFAIK so that shouldn't be a concern?

Reply:
True. I am OK to leave as-is.